slices/apache2.yaml

```diff
@@ -0,0 +1,164 @@
+      /var/cache/apache2/mod_cache_disk/:
+      /var/lib/apache2/conf/: {make: true, mode: 0755}
+      /var/lib/apache2/conf/enabled_by_maint/: {make: true, mode: 0755}
+      /var/lib/apache2/conf/enabled_by_maint/charset: {text: "", mode: 0644}
+      /var/lib/apache2/conf/enabled_by_maint/localized-error-pages: {text: "", mode: 0644}
+      /var/lib/apache2/conf/enabled_by_maint/other-vhosts-access-log: {text: "", mode: 0644}
+      /var/lib/apache2/conf/enabled_by_maint/security: {text: "", mode: 0644}
+      /var/lib/apache2/conf/enabled_by_maint/serve-cgi-bin: {text: "", mode: 0644}
+      /var/lib/apache2/module/: {make: true, mode: 0755}
+      /var/lib/apache2/module/enabled_by_maint/: {make: true, mode: 0755}
+      /var/lib/apache2/module/enabled_by_maint/access_compat: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/alias: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/auth_basic: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/authn_core: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/authn_file: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/authz_core: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/authz_host: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/authz_user: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/autoindex: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/deflate: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/dir: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/env: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/filter: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/mime: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/mpm_event: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/negotiation: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/reqtimeout: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/setenvif: {text: "", mode: 0644}
+      /var/lib/apache2/module/enabled_by_maint/status: {text: "", mode: 0644}
+      /var/lib/apache2/site/: {make: true, mode: 0755}
+      /var/lib/apache2/site/enabled_by_maint/: {make: true, mode: 0755}
+      /var/lib/apache2/site/enabled_by_maint/000-default: {text: "", mode: 0644}
+      /var/log/apache2/:
+      /var/log/apache2/access.log: {text: "", mode: 0640}
+      /var/log/apache2/error.log: {text: "", mode: 0640}
+      /var/log/apache2/other_vhosts_access.log: {text: "", mode: 0640}
+
+  copyright:
+    contents:
+      /usr/share/doc/apache2/copyright:
+
+  index:
+    essential:
+      - apache2-data_default-site
+      - apache2_documentroot
+    contents:
+      /var/www/html/index.html:
+        copy: /usr/share/apache2/default-site/index.html
```

Comment:
So it appears chisel doesn't like copy files from different packages.
Can we try something like this to see if it works?
```
    contents:
      /var/www/html/index.html: {text: FIXME, mutable: true}
    mutate: |
      index = content.read("/usr/share/apache2/default-site/index.html")
      content.write("/var/www/html/index.html", index)
```